ui-tests/tests/jupyterlab_new_launcher.spec.ts

```diff
@@ -80,3 +81,24 @@ test.describe('Filter individual', () => {
     );
   });
 });
+
+test.describe('Quick Settings', () => {
+  test('open quick settings menu', async ({ page }) => {
+    const launcher = page.locator('.jp-LauncherBody');
+    await page.locator('.jp-Launcher-QuickSettings').click();
+    expect(await launcher.screenshot()).toMatchSnapshot(
+      'launcher_open_quicksettings.png'
+    );
+  });
+
+  test('show starred from quick settings', async ({ page }) => {
+    const launcher = page.locator('.jp-LauncherBody');
+    await page.locator('.jp-Launcher-QuickSettings').click();
+    await page
+      .locator('.lm-Menu-itemLabel:text("Show Starred Section")')
+      .click();
+    expect(await launcher.screenshot()).toMatchSnapshot(
+      'launcher_show_starred_from_quicksettings.png'
+    );
```

Comment:
I would suggest to drop this screenshot comparison and instead add something like:

```js
const starredSection = page.locator('.jp-CollapsibleSection-Title:has-text("Starred")')
await expect(starredSection).toBeVisible();
```

(I have not tested these lines so they may need adjusting